scripts/xor-codec/hash_codec.py

```diff
@@ -0,0 +1,91 @@
+import os
+import sys
+import numpy
+import hashlib
+
+valid_hashes = [
+	# PyTorch Checkpoint V1:
+	"f856e9d99c30855d6ead4d00cc3a5573",
+	"d9dbfbea61309dc1e087f5081e98331a",
+	"2b2bed47912ceb828c0a37aac4b99073",
+	"ea0405cdb5bc638fee12de614f729ebc",
+	"4babdbd05b8923226a9e9622492054b6",
+	# PyTorch Checkpoint V2:
+	"d9dbfbea61309dc1e087f5081e98331a",
+	"f856e9d99c30855d6ead4d00cc3a5573",
+	"4babdbd05b8923226a9e9622492054b6",
+	"ea0405cdb5bc638fee12de614f729ebc",
+	"2b2bed47912ceb828c0a37aac4b99073",
+	# HuggingFace Checkpoint V1:
+	"d0e13331c103453e9e087d59dcf05432",
+	"29aae4d31a0a4fe6906353001341d493",
+	"b40838eb4e68e087b15b3d653ca1f5d7",
+	"f845ecc481cb92b8a0586c2ce288b828",
+	"f3b13d089840e6caf22cd6dd05b77ef0",
+	"12e0d2d7a9c00c4237b1b0143c48a05e",
+	"1348f7c8bb3ee4408b69305a10bdfafb",
```

Comment:
I am completely confused now. What are these hashes? they look like the ones we have in the readme as outputs of the LLaMA -> HF conversion process..

Reply:
The hashes from the readme were included verbatim in order to maintain compatibility for users ending up with these hashes.